tools/tracegen/TDFParser.cpp

```diff
@@ -135,7 +135,7 @@ TDFParser::parse()
 			goto failed;
 		}
 
-		strncpy(tdf->header.datfilename, DEFAULT_DAT_FILE, strlen(DEFAULT_DAT_FILE));
+		strcpy(tdf->header.datfilename, DEFAULT_DAT_FILE);
```

Comment:
were newer versions of GCC complaining about this?

Reply:
Yeah